commons/src/test/java/org/eclipse/kapua/commons/setting/AbstractKapuaSettingTest.java

```diff
@@ -67,5 +79,4 @@ protected TestSetting() {
         }
 
     }
-
-}
+}
```

Comment:
Please add tests for:
 - createCompositeSource
 - loadConfigResources
 - loadConfigResource
classes, as they are not covered with tests.